.gitignore

```diff
@@ -7,3 +7,5 @@ Cargo.lock
 #fuzz
 fuzz/hfuzz_target
 fuzz/hfuzz_workspace
+
+*~
```

Comment:
I should have complained earlier when the `.idea` stuff got added, but these editor-specific ignores would be better suited for `.git/info/exclude` since we don't have a blessed standard editor and don't want to add new ones all the time.

Reply:
I don't mind adding new ones all the time :P. Do you think we'll ever have a use for files ending in `~`?

In any case I also don't mind removing this.

Reply:
Well, then it's ok I guess ^^